mdsuite/calculators/green_kubo_viscosity.py

```diff
@@ -119,13 +132,17 @@ def _calculate_prefactor(self, species: str = None):
         -------
 
         """
-        # Calculate the prefactor
         # prepare the prefactor for the integral
         numerator = 1  # self.experiment.volume
-        denominator = 3 * (self.data_range - 1) * self.experiment.temperature * self.experiment.units[
-            'boltzman'] * self.experiment.volume  # we use boltzman constant in the units provided.
-        prefactor_units = self.experiment.units['pressure'] ** 2 * self.experiment.units['length'] ** 3 * \
-            self.experiment.units['time'] / self.experiment.units['energy']
+        denominator = 3 * (self.data_range - 1) * \
+                      self.experiment.temperature * \
+                      self.experiment.units['boltzman'] * \
+                      self.experiment.volume
+
+        prefactor_units = self.experiment.units['pressure'] ** 2 * \
+                          self.experiment.units['length'] ** 3 * \
+                          self.experiment.units['time'] / \
+                          self.experiment.units['energy']
```

Comment:
we should make this a dataclass or a named tuple:

```py
from dataclasses import dataclass

@dataclass(frozen=True)
class Units:
    pressure: float = 1.23456
    length: float = 1.1234656
    
   def gk_prefactor(self):
       return self.pressure ** 2 * self.length ** 3 + ...
```

so we can have `self.experiment.units.pressure` or `self.experiment.units.gk_prefactor()`

Reply:
Issue #229 might be a good place for this kind of discussion. I suggest there that we push globals to a data file but a class also sounds quite good. We should discuss the pros and cons. In general though this is a really good idea.